src/command_modules/azure-cli-consumption/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.2.3
++++++
+* Added new commands for marketplace.
```

Comment:
Need to add that certain subgroups have been renamed. Annotate this as a breaking change and bump the minor version.

Reply:
Not sure of the format of release history based on your comment. @tjprescott Let us know if this is OK.
0.2.3
+++++
* Added new commands for marketplace.
* Renamed commands for reservation summary and detail.
* Updated options list for reservation and pricesheet.
* Breaking change

Reply:
@tjprescott Please let me know if you have any comments on it.

Reply:
I would recommend:
```
* Added commands `...`
* BREAKING CHANGE: `reservations` commands renamed to `reservation`
* BRAKING CHANGE: `details` commands renamed to `detail`
* Added/Removed (?) options for ... (note: if you removed short options, this would be a breaking change too)
```

Reply:
@tjprescott All comments are resolved with new commits. Please review.